draft-ietf-quic-multipath.md

```diff
@@ -319,7 +319,11 @@ Path ID 0.
 After the handshake concluded if negotiation of multipath support succeeded,
 endpoints SHOULD use PATH_ACK frames instead of ACK frames,
 also for acknowledging so far unacknowledged 0-RTT packets, using
-Path ID 0. Similarly after a successful handshake, endpoints SHOULD also use
+Path ID 0. Further more, endpoints SHOULD use PATH_ACK frames in all 1-RTT packets after
+the successful negotiation for multipath. If endpoints receive ACK frame
```

Comment:
This additional sentence is not needed at this is already what the sentence above says. All packets after the handshake has concluded are 1-RTT packets.